packages/mobile/src/escrow/saga.ts

```diff
@@ -144,7 +144,7 @@ function* withdrawFromEscrow() {
     signature = signature.slice(2)
     const r = `0x${signature.slice(0, 64)}`
     const s = `0x${signature.slice(64, 128)}`
-    const v = web3.utils.hexToNumber(signature.slice(128, 130))
+    const v = web3.utils.hexToNumber(`0x${signature.slice(128, 130)}`)
```

Comment:
Nit: use the `ensureHexLeader` helper here
https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/src/address.ts#L49